src/App.test.js

```diff
@@ -364,7 +364,7 @@ describe('App', () => {
     });       
   });
 
-  describe.skip('Wave 4:  reset game button', () => {
+  describe('Wave 4:  reset game button', () => {
```

Comment:
It doesn't look like you implemented the optional reset functionality, so you could leave these skipped.